docs/mpdev-references.md

```diff
@@ -119,6 +119,23 @@ Properties in `/data-test/schema.yaml` will override properties in
 `/data/schema.yaml`. This can also be used to overwrite existing default
 values for verification.
 
+#### Verification logs
+
+Logs from verify can be extensive, so they are separated in multiple files to facilitate troubleshooting
+
+- **resources.yaml:** Output of `kubectl get` for different kind of resources in the namespace. It should help visualize the footprint of the application after installation is complete.
```

Comment:
nit: Can we wrap in ` for consistency with other filename references in the doc?